### ISSUE TITLE
I want the robot to move according to the target position, not by keyboard control, is there any good solution？？？

### ISSUE THREAD

**Issue by bestgodok** [Unity-Technologies/articulations-robot-demo#37]

**Comment by RomanStadlhuber** [comment]
I deactivated both Controller scripts in the root link, but then you have to set all Articulations Parameters for all Articulations Components yourself. 
This is how they set target rotation in their own source code: https://github.com/Unity-Technologies/articulations-robot-demo/blob/72d05339e625208c5e8e025bdca1fc2d2d3d308f/ArmRobot/Assets/Scripts/ArticulationJointController.cs#L44

**Comment by bestgodok** [comment]
> I deactivated both Controller scripts in the root link, but then you have to set all Articulations Parameters for all Articulations Components yourself.
> This is how they set target rotation in their own source code:
> 
> https://github.com/Unity-Technologies/articulations-robot-demo/blob/72d05339e625208c5e8e025bdca1fc2d2d3d308f/ArmRobot/Assets/Scripts/ArticulationJointController.cs#L44

First, thank you for your reply！So,do I have to use inverse kinematics to figure out the parameters of each joint?

**Comment by RomanStadlhuber** [comment]
Yes you do. The Assetstore / Community provide lots of IK solutions though not all methods fit equally well for all systems.
In case you don't already know of a specific solution you would be better off researching for one that fits your robot.

**Comment by bestgodok** [comment]
Thank you very much for your suggestion. I will try to solve it！
